### PR TITLE
Ts doggo

### PR DESCRIPTION
**Pull request by tkylesh**
Modified MVC Chapter from previous server-side curriculum to use .net 6, EF Core, VS Code, PostgreSQL, and dotnet-aspnet-codegenerator.

### CODE REVIEW

File: book-5-mvc/README.md [hunk 1]
```diff
@@ -1 +1,46 @@
-# Book 5 - Server-Side Rendering with ASP.NET MVC
\ No newline at end of file
+# Book 5 - Server-Side Rendering with ASP.NET MVC
+Congratulations, You've made it to book 5!  
+In this book you will learn a little bit about server-side rendering and the ASP.NET MVC framework.  
+
+Hold your horses...What is server-side rendering? You may be thinking.  Let's discuss.  We'll start by addressing the term "server-side". A server refers to a computer somewhere on the Internet, so when we say server-side, we're simply describing that it's happening on a computer somewhere on the Internet.  What about "redering"? In the context of a web application we use the word "render" to refer to the process of dynamically generating HTML using some programming language. MVC, which stands for Model View Controller, is a popular pattern for server-side rendering applications. ASP.NET Core MVC is a popular framework used to build server-side rendering applications.
```
Comment (joshdbarton) [inline book-5-mvc/README.md:5]: "redering" -> "rendering"

File: book-5-mvc/chapters/ProjectSetup.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+
+
+## Configuration
+You will need a connection string in order to create and modify your postgres database.  Let's do that now.
+
+1. Initialize user-secrets
+    - ```dotnet user-secrets init```
+1. Add your connection string
+    - ```dotnet user-secrets set DogGoDbConnectionString "Host=localhost;Database=DogGo;Username=postgres;Password=<your password>"```
```
Comment (joshdbarton) [inline book-5-mvc/chapters/ProjectSetup.md:39]: does this need a port?

**Review by joshdbarton — CHANGES_REQUESTED** [review]

File: book-5-mvc/README.md (hunk 1 above)
Comment (tkylesh) [reply book-5-mvc/README.md]: 👍

**Comment by joshdbarton** [comment]
:shipit:

**Review by joshdbarton — APPROVED** [review]

**Pull request merged**